ui/ui-components/components/session/SignIn.tsx

```diff
@@ -59,24 +66,69 @@ export default function SignInForm(props) {
       setProviders(response.providers);
     }
     setLoadingOauthProviders(false);
-  };
+  }, [execApi]);
```

Comment:
execApi is a hook... won't it not ever change in the mounted component?

Reply:
It was changing in my tests. I'm currently working on the execApi ticket so this will go away soon.